src/common/__tests__/cluster-store.test.ts

```diff
@@ -208,6 +210,12 @@ describe("config with existing clusters", () => {
     expect(storedClusters[1].preferences.terminalCWD).toBe("/foo2");
     expect(storedClusters[2].id).toBe("cluster3");
   });
+
+  it("marks owned cluster disabled by default", () => {
```

Comment:
enabled?

Reply:
No, disabled 🙂 If store is loaded from disk clusters with ownerRef will be disabled by default -> extension needs to mark them as enabled.